MAUI/Masked-Entry/Basic-Features.md

```diff
@@ -200,5 +200,88 @@ maskedEntry.TextColor =  Colors.Green;
 
 ![TextColor](MaskedEntry_Images/MaskedEntry_TextColor.png)
 
+## Setting Stroke
+
+The `Stroke` property is used to modify the visibility of border. The following code example demonstrates how to change the border visibility in [`SfMaskedEntry`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfMaskedEntry.html).
```

Comment:
please check the description of this title.

Reply:
Addressed the comments.